src/main/scala-2.11/markets/participants/strategies/LimitOrderTradingStrategy.scala

```diff
@@ -20,13 +20,13 @@ import akka.agent.Agent
 import markets.tickers.Tick
 import markets.tradables.Tradable
 
-import scala.collection.{immutable, mutable}
+import scala.collection.mutable
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/2168327519/issues/source?bid=2921361&fileBranchId=3162043#l23)